projects/js-packages/components/components/pricing-table/stories/index.stories.tsx

```diff
@@ -78,3 +110,90 @@ const DefaultArgs = {
 
 export const _default = Template.bind( {} );
 _default.args = DefaultArgs;
+
+const WithLogoTemplate: StoryFn< typeof PricingTable > = args => {
+	return (
+		<PricingTable { ...args }>
+			<PricingTableColumn primary>
+				<PricingTableHeader>
+					<ProductPrice
+						price={ 24.95 }
+						offPrice={ 12.48 }
+						legend="per month for the first year, then billed yearly"
+						currency="USD"
+						hidePriceFraction
+					/>
+					<Button fullWidth>Get Social</Button>
+				</PricingTableHeader>
+				<PricingTableItem isIncluded={ true } />
+				<PricingTableItem isIncluded={ true } />
+				<PricingTableItem isIncluded={ true } />
+				<PricingTableItem isIncluded={ true } />
+				<PricingTableItem isIncluded={ true } />
+				<PricingTableItem isIncluded={ true } />
+				<PricingTableItem isIncluded={ true } />
+				<PricingTableItem isIncluded={ true } />
+				<PricingTableItem isIncluded={ false } isComingSoon={ true } />
+			</PricingTableColumn>
+			<PricingTableColumn>
+				<PricingTableHeader>
+					<ProductPrice price={ 0 } legend="Free forever" currency="USD" hidePriceFraction />
+					<Button fullWidth variant="secondary">
+						Start for free
+					</Button>
+				</PricingTableHeader>
+				<PricingTableItem isIncluded={ false } />
+				<PricingTableItem isIncluded={ true } />
+				<PricingTableItem isIncluded={ true } />
+				<PricingTableItem isIncluded={ true } />
+				<PricingTableItem isIncluded={ true } />
+				<PricingTableItem isIncluded={ false } />
+				<PricingTableItem isIncluded={ false } />
+				<PricingTableItem isIncluded={ false } />
+				<PricingTableItem isIncluded={ false } />
+			</PricingTableColumn>
+		</PricingTable>
+	);
+};
+
+export const WithLogo = WithLogoTemplate.bind( {} );
+WithLogo.args = {
+	title: 'Write once, post everywhere',
+	headerLogo: <Logo height={ 32 } />,
```

Comment:
Let us use the text or move some logo inline for this story.
```suggestion
	headerLogo: 'Logo',
```

Reply:
Using a string defeats the purpose of the story where you can check how the component looks. Since it's really just a story file and nothing for production I'm not sure what we lose with referencing the logo from the other package, but if you really think it's better to copy it over to this file I can do it.

Reply:
I think copying it over is better. These cross-contaminations between packages have uneven consequences that are hard to debug.

Reply:
Okay, I'll copy it over